src/main/java/com/inin/analytics/elasticsearch/ESEmbededContainer.java

```diff
@@ -279,4 +287,46 @@ public void setNode(Node node) {
 		this.node = node;
 	}
 
+    public void setDefaultIndexSettings(Settings defaultIndexSettings) {
+        this.defaultIndexSettings = defaultIndexSettings;
+    }
+
+    public Settings getDefaultIndexSettings() {
+	    return defaultIndexSettings;
+	}
+
+    public ArrayList<Class<? extends Plugin>> getPluginClasses(String customPluginListFile) {
```

Comment:
Return a List (or some interface from the Collections framework) so we are working against interfaces.